### PR TITLE
Fix typo in plugin package

### PR DESCRIPTION
**Pull request by kanozec**
#### Summary

I just fix some typo in the plugin package.
#### Ticket Link

None

### CODE REVIEW

File: plugin/client_rpc.go [hunk 1]
```diff
@@ -525,8 +525,8 @@ func (s *hooksRPCServer) FileWillBeUploaded(args *Z_FileWillBeUploadedArgs, retu
 	return nil
 }
 
-// MessageWillBePosted is in this file because of the difficulty of identifiying which fields need special behaviour.
-// The special behaviour needed is decoding the returned post into the original one to avoid the unintentional removal
+// MessageWillBePosted is in this file because of the difficulty of identifying which fields need special behavior.
```
Comment (hanzei) [inline plugin/client_rpc.go]: :+1: identifiying/identifying, but behaviour is a valid spelling in BE

Comment (kanozec) [reply plugin/client_rpc.go]: Sorry, should I change "behavior" back to "behaviour" ??

Comment (hanzei) [reply plugin/client_rpc.go]: Yes please

Comment (hanzei) [reply plugin/client_rpc.go]: There are still two instances of `behavior`

**Comment by jasonblais** [comment]
Not sure if the automerge label should work in this case, since I don't have permissions to merge PRs

cc @hanzei

**Comment by jasonblais** [comment]
@cpanato the `AutoMerge` label is very cool. Is it okay for PMs (and other non core committers) without write access to have permissions to apply the label?

**Comment by cpanato** [comment]
@jasonblais yes, just add the label

**Pull request merged**